tools/hilbert.py

```diff
@@ -0,0 +1,589 @@
+#!/usr/bin/env python2
```

Comment:
well just like any script: e.g `./hilbert.py` - it will use Python2 by default. 
I can add a link named `hilbert` if `.py` should be avoided?